data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/server/RequestExceptionHandler.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.peerforwarder.server;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+
+import java.util.Objects;
+
+public class RequestExceptionHandler {
```

Comment:
Maybe rename this to clarify its purpose. It is for the server-side only as it handles responses.

Reply:
I named it RequestExceptionHandler because it handles exceptions caused by aggregated requests. But if something like `ResponseHandler` is more clear I'm okay with renaming it.